src/components/footer.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { containerMain } from "./../constants";
```

Comment:
huh ? this sounds weird ?

Reply:
not sure what to name, changed it to cssContainerWrapper.